docs/src/pages/components/toggle-button/toggle-button.md

```diff
@@ -21,6 +21,8 @@ With exclusive selection, selecting one option deselects any other.
 
 In this example, text justification toggle buttons present options for left, center, right, and fully justified text (disabled), with only one item available for selection at a time.
 
+**Note**: Exclusive selection does not enforce that at least one button must be active. For that effect see [enforce value set](#enforce-value-set).
```

Comment:
I'm not sure about "at least one", since with exclusive it can only be one (or none).
```suggestion
**Note**: Exclusive selection does not enforce that a button must be active. For that effect see [enforce value set](#enforce-value-set).
```